### ISSUE TITLE
Loss becomes nan at 19k steps

### ISSUE THREAD

**Issue by Hsn37** [TensorSpeech/TensorFlowTTS#754]
I am training tacotron2 from scratch on IPA symbols. the input is entirely based on IPA and the output is Urdu wav files.  I used your instructions and preprocessed my dataset according the ljspeech one, using the same format.

Setup:
Cuda: 11.2
cudnn: 8.1
GPU: RTX 3070
Driver: 510.54

The training started off smooth initially, and the loss is decreased steadily. The predictions and alignments also seemed to be evolving and getting better. However, after 19,200 steps, the model loss suddenly became nan. the logs did not show any extra information. So i could not tell what the reason was.

**At 19k steps:**
![5_alignment](https://user-images.githubusercontent.com/18398637/159922967-853fa783-5786-44ec-8d1b-12e37b78c478.png)
![15_alignment](https://user-images.githubusercontent.com/18398637/159923286-86a00768-9a53-4097-b948-9b3f40796283.png)
![b'c213'](https://user-images.githubusercontent.com/18398637/159923318-451f43e1-2a4e-4756-ab71-fdda85997cba.png)
![b'c412'](https://user-images.githubusercontent.com/18398637/159923345-bba73c6d-e719-4372-8aa6-687d3bf94f02.png)

then after this, the graphs are empty. nothing to be seen.
![b'c412'](https://user-images.githubusercontent.com/18398637/159923462-6a243c78-ffd5-4c2f-9646-39862dfd8337.png)
![11_alignment](https://user-images.githubusercontent.com/18398637/159923481-55d51717-af8d-49d4-a4ad-af4db0c9ed96.png)

Could you please help me with this and where the problem might be? and whether or not this is an issue on my end, or the library.
